dts/bindings/sensor/hzgrow,r502a.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2021, Linumiz
+# SPDX-License-Identifier: Apache-2.0
+
+description: HZ-Grow GROW_R502A Fingerprint sensor.
+
+compatible: "hzgrow,r502a"
+
+include: uart-device.yaml
+
+properties:
+    address:
+      type: int
+      required: false
+      default: 0xffffffff
```

Comment:
I'd prefer to remove `default` and just use DT_INST_PROP_OR() in the driver.

Reply:
I fixed this, and now I am getting this address via reg property by including uart-controller.yaml as per @gmarull suggestion.